### ISSUE TITLE
widget will not display when widget is configurable

### ISSUE THREAD

**Issue by blores** [clemensbartz/essential-launcher#71]
Multiple-entries widget that display rss feeds from spaRSS is not displayed. The simple version of widget works fine

Reproducable on two devices (both running LineageOS) android Pie and android7.

Seems like the version where it worked fine was already removed from F-droid.

**Comment by clemensbartz** [comment]
Which version of spaRSS are you using?

You can download all old release from [this site](https://github.com/clemensbartz/essential-launcher/releases). The oldest 1.x release is [1.61](https://github.com/clemensbartz/essential-launcher/releases/tag/v1.6.1).

Please note that the ones on Github are signed by myself, so you cannot update to the F-Droid version.

**Comment by blores** [comment]
1.61 installed from your link re-show/repopulate widget that is generated with spaRSS 1.11.8. 
Confirming for LineageOS Pie cheeseburger.

**Comment by clemensbartz** [comment]
I can reproduce the issue. Will take a look at it.

**Comment by clemensbartz** [comment]
This will take a little longer to investigate. When I am in debugging mode, in some instances the widget is shown, in others it is not. I have not found the cause for this, yet. I will keep you updated here.

**Comment by clemensbartz** [comment]
The root cause is that the launcher is trashed and reloaded when the configure activity of the widget is displayed. After returning, onStart() is performed again, but before the new widgetId is loaded:
```java
new LoadSharedPreferencesDAOTask(this, sharedPreferencesDAO, viewController, widgetController).execute();
```

This loads the widget which was set before (in many cases: none).

I have not found a fix, yet.

**Comment by clemensbartz** [comment]
Fixed and on track for 2.1.1.
